manim/mobject/opengl/opengl_mobject.py

```diff
@@ -1189,7 +1234,12 @@ def reverse(maybe_list):
         ]
 
         # Initialize row_heights / col_widths correctly using measurements as fallback
-        def init_sizes(sizes, num, measures, name):
+        def init_sizes(
+            sizes: list[float | None] | None,
+            num: int,
+            measures: list[float],
+            name: str,
+        ) -> list[float]:
```

Comment:
```suggestion
            sizes: Sequence[float | None] | None,
            num: int,
            measures: Mapping[float],
            name: str,
        ) -> Sequence[float]:
```
In general, typing stuff as `list[xyz]` is not the best idea because `list[OpenGLMobject]` is incompatible with `list[OpenGLVMobject]`